_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -0,0 +1,361 @@
+[tye paper]: https://arxiv.org/abs/2006.12031
+[harding mad miner]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-June/018010.html
+[zmnscpxj scorched earth]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-June/017998.html
+[scorched earth]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2013-May/002632.html
+[harding myopic]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-June/018009.html
+[news101 fee overpayment]: /en/newsletters/2020/06/10/#fee-overpayment-attack-on-multi-input-segwit-transactions
+[bcc11413rn]: https://github.com/kallewoof/bitcoin/blob/25dac9fa65243ca8db02df22f484039c08114401/doc/release-notes-11413.md
+[todd relay malleability]: https://petertodd.org/2016/segwit-consensus-critical-code-review#peer-to-peer-networking
+[news94 btcpayjoin]: /en/newsletters/2020/04/22/#btcpay-adds-support-for-sending-and-receiving-payjoined-payments
+[news13 bustapay]: /en/newsletters/2018/09/18/#bustapay-discussion
+[swambo rfc]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-June/017996.html
+[news87 negotiation]: /en/newsletters/2020/03/04/#improving-feature-negotiation-between-full-nodes-at-startup
+[mb ts]: https://diyhpl.us/wiki/transcripts/la-bitdevs/2020-05-21-alekos-filini-magical-bitcoin/
+[mb vid]: https://www.youtube.com/watch?v=QVhC2DOIl7I)
+[watchtowers ts]: https://diyhpl.us/wiki/transcripts/lightning-hack-day/2020-05-24-sergi-delgado-watchtowers/
+[watchtowers vid]: https://www.youtube.com/watch?v=Vkq9CVxMclE
+[watchtowers slides]: https://srgi.me/resources/slides/Potzblitz!2020-Watchtowers.pdf
+[coinswap ts]: https://diyhpl.us/wiki/transcripts/wasabi-research-club/2020-06-15-coinswap/
+[coinswap vid]: https://www.youtube.com/watch?v=Pqz7_Eqw9jM
+[socratic ts]: https://diyhpl.us/wiki/transcripts/london-bitcoin-devs/2020-06-16-socratic-seminar-bip-schnorr/
+[ruffing ts]: https://diyhpl.us/wiki/transcripts/london-bitcoin-devs/2020-06-17-tim-ruffing-schnorr-multisig/
+[socratic vid]: https://www.youtube.com/watch?v=uE3lLsf38O4
+[ruffing vid]: https://www.youtube.com/watch?v=8Op0Glp9Eoo
+[ruffing slides]: https://slides.com/real-or-random/taproot-and-schnorr-multisig
+[sydney ts]: https://diyhpl.us/wiki/transcripts/sydney-bitcoin-meetup/2020-06-23-socratic-seminar/
+[magical bitcoin]: https://magicalbitcoin.org/
+[news75 watchtower bolt]: /en/newsletters/2019/12/04/#proposed-watchtower-bolt
+[news98 sas]: /en/newsletters/2020/05/20/#two-transaction-cross-chain-atomic-swap-or-same-chain-coinswap
+[teos]: https://github.com/talaia-labs/python-teos
+[news74 wagner]: /en/newsletters/2019/11/27/#schnorr-taproot-updates
+[news100 coinswap]: /en/newsletters/2020/06/03/#design-for-a-coinswap-implementation
+[cpp compiler support]: https://en.cppreference.com/w/cpp/compiler_support#C.2B.2B17_features
+[potzblitz]: https://www.youtube.com/playlist?list=PLwgam6YBS0-jk1TlXD7QXDjTYJh-eJn_X
+[knw paper]: https://arxiv.org/pdf/2006.08513.pdf
```

Comment:
Incorrect link. Should be https://eprint.iacr.org/2020/774

Reply:
Whoops, too many tabs open with papers to read!